### DISCUSSION TITLE
how to connect sqlite3 with cipher type : WxSQLite3 ,Cipher : system.data.rc4 encrypted database using java or else remove the password for database

### DISCUSSION THREAD

**Discussion by ShadowTechnology** [pawelsalawa/sqlitestudio#4858] [General]
I already have Sqlite3 database with cipher type : WxSQLite3 ,Cipher : system.data.rc4 encrypted and know i want to connect this database using java and run the table i wanted. but when i tried to connect this database it says File opened that is not a database file error occurs. Or remove the password for my database. and i attached the code and database file in zip file to check the code is correct or not
[Databasezip.zip](https://github.com/pawelsalawa/sqlitestudio/files/12438747/Databasezip.zip)

**Comment by pawelsalawa** [comment]
Execute `PRAGMA rekey = '';` in SQLiteStudio on the encrypted database. This should remove encryption. You should be able to open it now in your Java app.

Keep in mind that SQLiteStudio will still try to open it using configured password, so best way is to remove the database from SQLiteStudio and re-add it using regular "SQLite 3" driver, so without any encryption.

**Comment by ShadowTechnology** [comment]
Thanks Pawelsalawa
